debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/memory/MemoryLogMinerEventProcessor.java

```diff
@@ -215,6 +215,14 @@ protected void handleCommit(LogMinerEventRow row) throws InterruptedException {
             return;
         }
 
+        if (transaction.getUserName() == null && !transaction.getEvents().isEmpty()) {
+            LOGGER.warn("Got transaction with null username {}", transaction);
```

Comment:
I don't think we need to necessarily issue a _warn_ for this use case do you @zxpzlp ?